common.mk

```diff
@@ -125,6 +125,10 @@ PRODUCT_PACKAGES += \
 PRODUCT_COPY_FILES += \
     $(COMMON_PATH)/rootdir/vendor/etc/apns-conf.xml:system/etc/apns-conf.xml
 
+MODEM_CONFIG := $(shell find $(COMMON_PATH)/rootdir/vendor/oem -type f -printf '%p:$(TARGET_COPY_OUT_VENDOR)/oem/%P\n')
```

Comment:
ifeq MODEM_CONFIG_LEGACY_PLATFORM <=== Loire Tone Yoshino Nile
MODEM_CONFIG := $(shell find $(COMMON_PATH)/rootdir/vendor/oem/modem-config-legacy  BLAHBLAHBLAH)
else <==== NOT LEGACY: Ganges Tama Kumano Seine && newer
MODEM_CONFIG := whatever you added already
endif